### DISCUSSION TITLE
Issue templating in .github repository

### DISCUSSION THREAD

**Discussion by bouteillerAlan** [community/community#53010] [Repositories]
### Select Topic Area

Question

### Body

Hello, I have a `.github` repository like that :
```
.
├── FUNDING.yml
├── ISSUE_TEMPLATE
│   ├── 1-bug-report.yml
│   ├── 2-feature-request.yml
│   └── config.yml
└── PULL_REQUEST_TEMPLATE.md 
```

But in my another repository, [let's say this one for example](https://github.com/bouteillerAlan/blurredwallpaper), went I click on new issue button I don't have any of my template. Same for the `PULL_REQUEST_TEMPLATE`.

On the other hand the `FUNDING` file work fine.

But if I create a `.github` folder directly into the repository, everything work fine.

What I do wrong ?

**Comment by SMelidoni** [comment]
Hi @bouteillerAlan 👋🏽

It seems that the issue is arising from the fact that you are using a centralised `.github` repository to store issue templates and pull request templates for multiple repositories. While this method works for the `FUNDING.yml` file, it does not work for issue templates and pull request templates.

To make issue templates and pull request templates work, you need to place the `.github` folder with the `ISSUE_TEMPLATE` folder and `PULL_REQUEST_TEMPLATE.md` file directly in each of your repositories where you want to use them.

So for each repository where you want to use the templates, your folder structure should look like this:

```
└── .github
    ├── ISSUE_TEMPLATE
    │   ├── 1-bug-report.yml
    │   ├── 2-feature-request.yml
    │   └── config.yml
    └── PULL_REQUEST_TEMPLATE.md
```

This should make your issue and pull request templates work as expected.

**Comment by bouteillerAlan** [comment]
It takes away a big interest in having a global `.github` repo :c 

On the other hand, I have the impression that this page says the opposite, no?
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file

![image](https://user-images.githubusercontent.com/30547569/232465113-c22c47e2-7852-4b3c-a0ee-a5b9e345c9eb.png)

**Comment by SMelidoni** [comment]
I see, I apologise for the confusion. It seems like you have already done what has been suggested by the documentation you've referenced. I guess the only other thing I might recommend to try and fix the issue is:

- Make sure the `.github` repository is public, as private repositories will not share their default community health files with other repositories.

**Reply by bouteillerAlan** [reply]
no problem :smile: 
yes this is the case https://github.com/bouteillerAlan/.github

**Comment by Certseeds** [comment]
@bouteillerAlan Is this problem solved? Now it seems that the Issue_template in the .github repo still has not played a role (even new a Issue_template in the .github repo itself)
